Assets/Scripts/ManagerScripts/CountdownManager.cs

```diff
@@ -9,38 +9,55 @@ public class CountdownManager: MonoBehaviour
         public TMP_Text countdownText;
         public GameObject countdownUI;
         public AudioSource countdownSound;
-        private bool _shouldCountTime;
+        public bool shouldCountTime;
         private bool _soundPlayed;
+        private bool _gameStart;
+        
 
         private void Start()
         {
-            _shouldCountTime = SceneManager.GetActiveScene().name != "MainMenuScene";
+            shouldCountTime = SceneManager.GetActiveScene().name != "MainMenuScene";
             countdown = 5f;
+            _gameStart = true;
         }
         
         private void Update()
         {
-            if ( _shouldCountTime && !_soundPlayed)
+            if ( shouldCountTime && !_soundPlayed)
             {
                 countdownSound.Play();
                 _soundPlayed = true;
             }
             
-            if (_shouldCountTime && countdown > 0)
+            if (shouldCountTime && countdown > 0)
             {
-                countdown -= Time.deltaTime;
+                if (_gameStart) SubstractTime();
```

Comment:
Minor change: I think instead of this if/else, we can put SubtractTime(_gameStart) here, and make a method like this:
private void SubstractTime(bool gameStart) {
    countdown -= gameStart ? Time.deltaTime : Time.unscaledDeltaTime;
}

I think it's a little cleaner this way since we're gonna have 3 state variables once the "Pausing during countdown screws things up" PR is merged